cypress/fixtures/analysis.json

```diff
@@ -244,5 +244,12 @@
     "source_analysis_on_pet_clinic_app": {
         "source": "Source code",
         "appName": "pet-clinic"
+    },
+    "upload_binary_analysis_on_jee_app": {
+        "source": "Upload a local binary",
+        "target": ["Application server migration"],
```

Comment:
according to bug 2001 , no target is set and tackle test app , source+dep analysis . 


Analyze tackle-test app with the following analysis configuration:

    Source code + dependencies
    Skip "Set targets"
    Application and all dependencies, including known Open Source libraries
    Upload manual rules file:
    No additional advanced options